CHANGELOG.md

```diff
@@ -74,7 +74,9 @@
 
 #### Bug Fixes
 
-* None.
+* Fix false positives in `inclusive_language` rule.  
```

Comment:
Technically, "mastercard" wasn't a false positive but rather just a commonly used name that we're special-casing it here. I'd also say this is less of a bug fix and more of an enhancement.

I'll push up some changes to the changelog to reflect this.